scripts/jenkins/build_docker.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+if ! command -v docker
+then
+  echo "ERROR: Building Docker image requires Docker binary to be installed" && exit 1
+elif ! docker version
+then
+  echo "ERROR: Building Docker image requires Docker daemon to be running" && exit 1
+elif ! command -v curl
+then
+  echo "ERROR: Building Docker image requires cURL to be installed" && exit 1
+fi
+
+echo "INFO: Determining latest tag"
+if [ ! -z ${TAG_NAME+x} ]
+then
+  echo "INFO: Detected TAG_NAME variable. Probably a Jenkins instance."
+  readonly GIT_TAG_DEFAULT=$(echo $TAG_NAME|sed s/^v//)
+else
+  echo "INFO: Did not detect TAG_NAME. Examining git log for latest tag"
+  readonly GIT_TAG_DEFAULT=$(git describe --abbrev=0|sed s/^v//)
+fi
+
+readonly GIT_TAG=${GIT_TAG:-$GIT_TAG_DEFAULT}
+
+readonly SCRIPT_PATH="$( cd "$(dirname "$0")" ; pwd -P )"
+readonly PROJECT_ROOT=$SCRIPT_PATH/../../
+readonly NAMESPACE="observability"
+
+if [ "$(ls -A  $SCRIPT_PATH/../../elastic-apm-agent/target)" ]
+then
+  # We have build files to use
+  echo "INFO: Found local build artifact. Using locally built for Docker build"
+  find -E $PROJECT_ROOT/elastic-apm-agent/target -regex '.*/elastic-apm-agent-[0-9]+.[0-9]+.[0-9]+(-SNAPSHOT)?.jar' -exec cp {} $PROJECT_ROOT/apm-agent-java.jar \; || echo "INFO: No locally built image found"
```

Comment:
PROJECT_ROOT ends with `/`

```suggestion
  find -E ${PROJECT_ROOT}elastic-apm-agent/target -regex '.*/elastic-apm-agent-[0-9]+.[0-9]+.[0-9]+(-SNAPSHOT)?.jar' -exec cp {} ${PROJECT_ROOT}apm-agent-java.jar \; || echo "INFO: No locally built image found"
```